frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -381,6 +381,7 @@ class AccountsSummary extends Component<Props, State> {
             text: t('guide.accountSummaryAmount.text'),
             title: t('guide.accountSummaryAmount.title')
           }} />
+          <Entry key="trackingModePortfolioChartMobile" entry={t('guide.trackingModePortfolioChartMobile')} />
```

Comment:
Nit: now that the guide entry is for both mobile and desktop remove "Mobile" from the key and rename to `trackingModePortfolioChart` or similar?

Reply:
Oh definitely makes sense, forgot to remove it actually. Thanks!